docs/content/Caching/Using-Pre-Aggregations.md

```diff
@@ -123,21 +123,46 @@ strategy has certain limitations:
 We highly recommend leaving `readOnly` unset or explicitly setting it to
 `false`.
 
+## Partitioning
+
+[Partitioning][wiki-partitioning] is an extremely effective optimization for
+improving data access. It effectively "shards" the data between multiple tables,
+splitting them by a defined attribute. An incoming query would be checked for
+this attribute, and any valid partitions required to satisfy it are selected.
```

Comment:
Will it work better if we say that "...unnecessary data won't be scanned and processed" (and that's why the reduced refresh time and all)?

Reply:
I've changed it to the following:

```
An incoming query would be checked for
this attribute, and **only** valid partitions required to satisfy it are
selected. This results in faster refresh times due to unnecessary data not being
scanned and processed, and possibly even reduced cost, depending on your
database solution.
```